### PR TITLE
Update the dark variation typography

### PR DESCRIPTION
**Pull request by m1r0**
#### Changes proposed in this Pull Request:
* Update the typography for the blue variation of the theme.

Testing instructions:
* Checkout the `feature/learning-mode-improvements` branch for Sensei LMS.
* Go to Appearance -> Themes and activate the Course theme.
* Go to Appearance -> Editor -> Templates -> Lesson (Learning Mode - Default).
* Click Styles -> Browse Styles -> Dark, click Save.
* Compare the typography with the design. Use an HTML vs Design tool ([example](https://chrome.google.com/webstore/detail/pixelparallel-by-htmlburg/iffnoibnepbcloaaagchjonfplimpkob?hl=en)) because the CSS provided by the Figma design is not always accurate (mainly for letter spacing).

#### Related issue(s):
Resolves: https://github.com/Automattic/themes/issues/7124

#### HTML vs Design:

Course title:

Progress:

Course navigation:

Module title:

Post title:

Content:

Actions:

Pagination:

### CODE REVIEW

**Comment by Imran92** [comment]
Looks good overall, found the following small issues

Desktop:
The lesson summary probably needs a bit of change
Screen Shot 2023-05-09 at 6 20 47 PM
The module title looks different
Screen Shot 2023-05-09 at 6 21 23 PM

Mobile:
The Exit course link looks a bit different
Screen Shot 2023-05-09 at 6 24 37 PM
The lesson summary looks a little different
Screen Shot 2023-05-09 at 6 25 26 PM

**Comment by merkushin** [comment]
Noticed a few differences on the front end (desktop)

Pagination uses another font:
CleanShot 2023-05-09 at 15 56 47@2x

Letter spacing for module name in the course outline:
CleanShot 2023-05-09 at 15 58 21@2x

As I noticed [for the blue variation](https://github.com/Automattic/themes/pull/7046#issuecomment-1539906323), here I feel the same, font should be just a bit thicker.
CleanShot 2023-05-09 at 15 57 13@2x

**Comment by m1r0** [comment]
Thanks for the reviews!

@Imran92 The course navigation should be fixed by https://github.com/Automattic/themes/pull/7065/commits/3bdbed7b3499c277e6dc372f23195936a139d500. The exit course link should be fixed by https://github.com/Automattic/themes/pull/7065/commits/f0259e196ad0920561d2b6ac7755d622212bc64f.

@merkushin The pagination was fixed in https://github.com/Automattic/themes/pull/7065/commits/dd1b6319afa20ef49d2f5623b82646ed0918e1c4 but the number `3` page still looks out of place. Also, the font size looks different when you do an HTML-vs-design comparison. I'm guessing it's a font quirk. 🤷  Let me know if you can find a way to make it look better. The module name should be fixed by https://github.com/Automattic/themes/pull/7065/commits/3bdbed7b3499c277e6dc372f23195936a139d500.

About the font-weight issue, I'm not sure why you see it that way. It looks ok to me. 🤷  Let me know if you want to do a screen share to figure this out.

**Comment by Imran92** [comment]
Thanks for the updates @m1r0 ! For some reason the exit course still didn't work for me. I've looked into it and it looks like a media query takes precedence over the dark.json setting and makes the font bigger.

PixelParallel
image

CSS (from 782px to lower)
image

File: course/styles/dark.json [hunk 1]
```diff
@@ -199,11 +207,61 @@
...
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"fontSize": "clamp(1.125rem, 1.125rem, 1.125rem)",
+					"letterSpacing": "0"
+				}
+			},
+			"sensei-lms/page-actions": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"fontSize": "clamp(1.125rem, 1.125rem, 1.125rem)",
```
Comment (Imran92) [inline course/styles/dark.json]: As all units are the same 1.125rem, do we need the clamp here?

File: course/styles/dark.json [hunk 2]
```diff
@@ -199,11 +207,61 @@
...
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--small)",
+					"fontWeight": "500",
+					"letterSpacing": "-0.01em"
+				}
+			},
+			"sensei-lms/course-theme-course-progress-counter" : {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--heading)",
+					"fontSize": "clamp(1.125rem, 1.125rem, 1.125rem)",
```
Comment (Imran92) [inline course/styles/dark.json]: Same as above

File: course/styles/dark.json (hunk 1 above)
Comment (m1r0) [reply course/styles/dark.json]: I've tried but it gets converted to `clamp(0.875rem, 0.875rem + ((1vw - 0.48rem) * 0.481), 1.125rem)`  automatically. 😞

Comment (Imran92) [reply course/styles/dark.json]: I see that 1.125rem is used in multiple places (I've found 3), maybe you can consider adding a custom fontsize like this
image

And then use that fontsize using `var(--wp--custom--typography--font-sizes--<Your-Font-Size-Name>)`

I tried it locally and it worked for me.

Comment (m1r0) [reply course/styles/dark.json]: Makes sense. I've added a new custom font size in 731aaa13b417c9063b65a9ad3b5d378ceb44416f. 👍

**Review by Imran92 — APPROVED** [review]
LGTM 👍

**Review by Imran92 — CHANGES_REQUESTED** [review]
Oh just noticed this, This font should be Syne, but it's Inter

Screen Shot 2023-05-15 at 2 59 13 PM

**Comment by m1r0** [comment]
Oh, good catch. Fixed in https://github.com/Automattic/themes/pull/7065/commits/258cecb6bfe0b054d0ea93022c6b653e24de0e2c.

**Pull request merged**